src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/fc_bias_fusion.cpp

```diff
@@ -74,3 +75,80 @@ ov::intel_cpu::FullyConnectedBiasFusion::FullyConnectedBiasFusion() {
     auto m = std::make_shared<ngraph::pattern::Matcher>(m_add, matcher_name);
     this->register_matcher(m, callback);
 }
+
+//CPU plugin would config LPT  not to propogate dequantization scale over bias to follow ONEDNN 3.x scheme.
+//It is a little tricky now to first fuse bias not DQ for pattern "FC + DQ + BIAS".
+//todo: Will move the FullyConnnect fusing into CPU and fuse the DQ and BIAS in topology order.
+ov::intel_cpu::QuantizedFullyConnectedBiasFusion::QuantizedFullyConnectedBiasFusion() {
+    MATCHER_SCOPE(FullyConnectedBiasFusion);
+    auto input = ngraph::pattern::any_input();
+    auto weights = ngraph::pattern::any_input(ngraph::pattern::has_static_shape());
+    auto m_fc = ngraph::pattern::wrap_type<ov::intel_cpu::FullyConnectedNode>({ input, weights }, [](ngraph::Output<ngraph::Node> output) {
+        return ngraph::pattern::consumers_count(1)(output) && ngraph::pattern::has_static_rank()(output);
+    });
+
+    auto m_scale = ngraph::pattern::any_input(ngraph::pattern::has_static_shape());
+    auto m_mul = ngraph::pattern::wrap_type<ngraph::opset1::Multiply>({m_fc, m_scale});
+
+    auto m_bias = ngraph::pattern::any_input(ngraph::pattern::has_static_shape());
+    auto m_add = ngraph::pattern::wrap_type<ngraph::opset1::Add>({m_mul, m_bias});
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher &m) {
+        auto& pattern_to_output = m.get_pattern_value_map();
+        auto mul = pattern_to_output[m_mul].get_node_shared_ptr();
+        auto scale = pattern_to_output[m_scale].get_node_shared_ptr();
+        auto add = pattern_to_output[m_add].get_node_shared_ptr();
+        auto bias = pattern_to_output[m_bias].get_node_shared_ptr();
+        auto fc = std::dynamic_pointer_cast<ov::intel_cpu::FullyConnectedNode>(pattern_to_output[m_fc].get_node_shared_ptr());
+        if (!fc || transformation_callback(fc)) {
+            return false;
+        }
+        //If the add is marked as BiasAttribute by LPT in FC + Multiply + add pattern,
+        //try fusing bias into FC.
+        if (!marked_as_bias(add))
```

Comment:
Sorry, I missed one more point.
I don't think that it's good idea to use LPT rt_info after LPT passes: at least, we have a request to delete all LPT rt info after LPT passes. If it happens, this code will not work.
I think, we can just duplicate the check from [MarkupBias](https://github.com/openvinotoolkit/openvino/blob/master/src/common/low_precision_transformations/src/markup_bias.cpp#L30) here

Reply:
OK. Will update. Thx for the information. BTW, would the rtinfo  out of LPT would be also cleaned? Or it means CPU plugin can't use the any ngraph node rtinfo ?

Reply:
In general it is not recommended to use rt_info for data propagation between components (like CPU and LPT) (excluding specific cases)
Especially because it is not safe in terms of generic transformation pipeline. Smb can enable additional transformation/fusing which might invalidate rt_info from previous transformation components.

Reply:
Thx! Just push commit to remove the check and seems  using the original bias add checking should be okay.

Reply:
I suppose it will be more safety if we also duplicate a check from MarkupBias for this bias. I mean the check on per-channel/per-tensor constant. Please see my first comment:
_I think, we can just duplicate the check from [MarkupBias](https://github.com/openvinotoolkit/openvino/blob/master/src/common/low_precision_transformations/src/markup_bias.cpp#L30) here_

Reply:
Applied the check. Thx!